eng/imports/HostAgnostic.props

```diff
@@ -32,6 +32,8 @@
     <PackageReference Include="Microsoft.VisualStudio.Threading" />
     <PackageReference Include="Microsoft.VisualStudio.Validation" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" />
+    
+    <PackageReference Include="Microsoft.IO.Redist" Condition="'$(TargetFramework)' == 'net472'" />
```

Comment:
so i did this in [PR](https://github.com/dotnet/project-system/pull/9504) but the Bug was not actually resolved because I forgot to do the package reference. I don't think there are others like this...